BezierKit/MacDemos/Demos.swift

```diff
@@ -466,6 +466,44 @@ class Demos {
                                 }
 
     })
+    static let demo23 = Demo(title: "PathComponent(arcCenter: ...)",
+                             quadraticControlPoints: [],
+                             cubicControlPoints: [],
+                             drawFunction: {(context: CGContext, demoState: DemoState) in
+                                if demoState.quadratic {
+                                    return
+                                }
+                                
+                                let pathComponent = PathComponent(
+                                    arcCenter: CGPoint(x: 100, y: 105),
+                                    radius: 100,
+                                    startAngle: .pi / 2,
+                                    endAngle: 2 * .pi,
+                                    clockwise: true
+                                )
+                                Draw.drawPathComponent(context, pathComponent: pathComponent)
+                                for curve in pathComponent.curves {
+                                    Draw.drawSkeleton(context, curve: curve)
+                                }
+    })
```

Comment:
let's try to make this demo more interactive. Maybe we can drag the endpoints to set the start and end angle?